records_mover/db/vertica/vertica_db_driver.py

```diff
@@ -104,6 +79,16 @@ def schema_sql(self, schema: str, table: str) -> str:
             # maybe a permission error?
             return super().schema_sql(schema, table)
 
+    # TODO: next 5 methods should be in loader - why aren't tests teling me that?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker